Frends.Kafka.Produce/Frends.Kafka.Produce/Definitions/Result.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace Frends.Kafka.Produce.Definitions;
+
+/// <summary>
+/// Task's result.
+/// </summary>
+public class Result
+{
+    /// <summary>
+    /// Message sent.
```

Comment:
This could be something more expanded, e.g. like MS docs - "True if message was produced successfully, false otherwise." Or something similar.